src/connect.js

```diff
@@ -19,8 +19,14 @@
  * @param {Object} baseParams
  * @return {Function}
  */
-function connect(params) {
-  // write code here
+function connect(params = {}) {
+  return (args = {}) => {
```

Comment:
good, but you can write simply
```suggestion
  return (args = {}) => { ...params, ... args };
```